src/m365/planner/commands/task/task-get.ts

```diff
@@ -168,6 +185,15 @@ class PlannerTaskGetCommand extends GraphCommand {
       });
   }
 
+  public optionSets(): string[][] | undefined {
+    return [
+      ['id', 'title'],
+      ['title', 'bucketId', 'bucketName'],
+      ['bucketName', 'planId', 'planName'],
+      ['planName', 'ownerGroupId', 'ownerGroupName']
```

Comment:
I think this will not this will introduce a bug. Probably it's my fault when I commented this on last review but I wanted only to add optionSet only for id and Title. We may use it for options where we need to specify at least one option but not both. In case were all options are optional it will not work as it will require the user to provide at least one. In our case if we use `--id` of the task we should not have to specify the bucket name or plan name etc. but after this change now we have
![image](https://user-images.githubusercontent.com/58668583/169913636-f1747635-6475-4e99-bd86-89e6d4c2687f.png)

do you think you could correct this one 🙂?

Reply:
so we may use optionSeti for `--id` and `--title` but we need to leave standard validation approach for resto of the options

Reply:
Oops.. my bad. I will fix it.

Reply:
double checked. Works like a charm ✅